### PR TITLE
Fix partition filter not working for `show segment`

### PR DESCRIPTION
**Pull request by congqixia**
/kind improvement

### CODE REVIEW

File: states/etcd/show/partition.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ func (rs *Partitions) PrintAs(format framework.Format) string {
 		for _, partition := range rs.Data {
 			fmt.Fprintf(sb, "Parition ID: %d\tName: %s\tState: %s\n", partition.ID, partition.Name, partition.State.String())
 		}
-		fmt.Fprintf(sb, "--- Total Database(s): %d\n", len(rs.Data))
+		fmt.Fprintf(sb, "--- Total Partition(s): %d\n", len(rs.Data))
```
Comment (yah01) [inline states/etcd/show/partition.go:48]: why name `Partitions` as `rs`?

Comment (yah01) [inline states/etcd/show/partition.go:48]: why name `Partitions` as `rs`?

**Review by yah01 — APPROVED** [review]
/lgtm

File: states/etcd/show/partition.go [hunk 2]
```diff
@@ -45,7 +45,7 @@ func (rs *Partitions) PrintAs(format framework.Format) string {
 		for _, partition := range rs.Data {
 			fmt.Fprintf(sb, "Parition ID: %d\tName: %s\tState: %s\n", partition.ID, partition.Name, partition.State.String())
 		}
-		fmt.Fprintf(sb, "--- Total Database(s): %d\n", len(rs.Data))
```
Comment (linetterobinson01-design) [inline states/etcd/show/partition.go]: Thank you

**Pull request merged**